modules/openapi-generator/src/main/resources/swift5/APIs.mustache

```diff
@@ -65,6 +65,32 @@ import Vapor
         return requestTask
     }
 
+    {{#useAsyncAwait}}
+    @discardableResult
+    {{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}open{{/nonPublicApi}} func execute() async throws -> Response<T> {
```

Comment:
Here we should have an `@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)` before the function.

Reply:
I fixed it!